test/local/metrics_tests.js

```diff
@@ -0,0 +1,362 @@
+          args = mocks.this.countAccountsWithMobileDevice.getCall(0).args
+          t.equal(args.length, 2, 'this.countAccountsWithMobileDevice was passed two arguments')
+          t.equal(args[0], 'fake db', 'this.countAccountsWithMobileDevice db argument was correct')
+          t.equal(args[1], Date.UTC(1977, 5, 10, 0, 0, 0, 0), 'this.countAccountsWithMobileDevice lastMidnight argument was correct')
+
+          t.equal(mocks.require.fs.appendFileSync.callCount, 1, 'fs.appendFileSync was called once')
+          args = mocks.require.fs.appendFileSync.getCall(0).args
+          t.equal(args.length, 2, 'fs.appendFileSync was passed two arguments')
+          t.equal(args[0], '/media/ephemeral0/fxa-admin/basic_metrics.log', 'log file path was correct')
+          var data = JSON.parse(args[1])
+          t.equal(Object.keys(data).length, 10, 'log file data had correct number of properties')
+          t.equal(data.hostname, 'fake hostname', 'log file hostname property was correct')
+          t.equal(data.pid, process.pid, 'log file pid property was correct')
+          t.equal(data.op, 'account_totals', 'log file op property was correct')
+          t.equal(data.total_accounts, 1, 'log file total_accounts property was correct')
+          t.equal(data.total_verified_accounts, 2, 'log file total_verified_accounts property was correct')
+          t.equal(data.total_accounts_with_two_or_more_devices, 3, 'log file total_accounts_with_two_or_more_devices property was correct')
+          t.equal(data.total_accounts_with_three_or_more_devices, 4, 'log file total_accounts_with_three_or_more_devices property was correct')
+          t.equal(data.total_accounts_with_mobile_device, 5, 'log file total_accounts_with_mobile_device property was correct')
+          t.equal(typeof data.time, 'string', 'log file time property was a string')
+          var time = new Date(data.time)
+          t.equal(time.getUTCFullYear(), 1977, 'log file time property had correct year')
+          t.equal(time.getUTCMonth(), 5, 'log file time property had correct month')
+          t.equal(time.getUTCDate(), 10, 'log file time property had correct date')
+          t.equal(time.getUTCHours(), 0, 'log file time property had correct hour')
+          t.equal(time.getUTCMinutes(), 0, 'log file time property had correct minute')
+          t.equal(time.getUTCSeconds(), 0, 'log file time property had correct second')
+          t.equal(time.getUTCMilliseconds(), 0, 'log file time property had correct millisecond')
+          t.equal(data.v, 0, 'log file v property was correct')
+          t.end()
+        })
+        .catch(function () {
+          t.fail('no errors should have occurred')
+          t.end()
+        })
+      }
+    )
+
+    test(
+      'teardown',
+      function (t) {
+        return db.close()
+      }
+    )
+
+    function createAccount (uid, time, emailVerified) {
```

Comment:
I can refactor this to use Danny's new helper from `fxa-auth-db-server/test/backend/db_tests.js`. I'll do that tomorrow.